File: Modul-4/Soal-5/PRAK405-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
int main(){
    int n, kelipatan;

    printf("Input baris pertama: ");
    scanf("%d", &n);
    printf("Input baris kedua: ");
    scanf("%d", &kelipatan);

    int total = 0;

    for (int j = 1; j <= n; j++)
    {
        int jumlah = 0;
        for (int k = j; k >= 1; k--)
        {
            int nilai = k * kelipatan;
            jumlah += nilai;
            printf("(%d * %d)", k, kelipatan);
            if (k > 1){
                printf(" + ");
            }
        }
        total += jumlah;
        printf(" = %d\n", jumlah);
    }
    printf("%d\n", total);
    return 0;
}

File: Modul-4/Soal-5/PRAK405-2410817220012-NOOR-KHALISA.py
n = int(input("Input baris pertama: "))
kelipatan = int(input("Input baris kedua: "))

total = 0

for j in range(1, n + 1):
    jumlah = 0

    for k in range(j, 0, -1):
        nilai = k * kelipatan
        jumlah += nilai
        print(f"({k} * {kelipatan})", end="")
        if k > 1:
            print(" + ", end="")

    total += jumlah
    print(f" = {jumlah}")

print(total)